### ISSUE TITLE
TypeError: Cannot read property 'length' of undefined

### ISSUE THREAD

**Issue by chrisgorgo** [OpenNeuroOrg/openneuro#671]
```
TypeError: Cannot read property 'length' of undefined
    at l (app-4d388932.bundle.js:56)
    at t.default (app-4d388932.bundle.js:56)
    at t.value (app-4d388932.bundle.js:51)
    at g (vendor-4d388932.bundle.js:30)
    at beginWork (vendor-4d388932.bundle.js:30)
    at a (vendor-4d388932.bundle.js:30)
    at e (vendor-4d388932.bundle.js:30)
    at N (vendor-4d388932.bundle.js:30)
    at n (vendor-4d388932.bundle.js:30)
    at O (vendor-4d388932.bundle.js:30)
    at y (vendor-4d388932.bundle.js:30)
    at h (vendor-4d388932.bundle.js:30)
    at Object.enqueueSetState (vendor-4d388932.bundle.js:30)
    at t.i.setState (vendor-4d388932.bundle.js:23)
    at t.a (app-4d388932.bundle.js:23)
    at a (app-4d388932.bundle.js:19)
    at a.emit (app-4d388932.bundle.js:23)
    at t.trigger (app-4d388932.bundle.js:19)
    at t.update (app-4d388932.bundle.js:1)
    at app-4d388932.bundle.js:1
    at app-4d388932.bundle.js:6
tn @ vendor-4d388932.bundle.js:30
commitErrorLogging @ vendor-4d388932.bundle.js:30
M @ vendor-4d388932.bundle.js:30
N @ vendor-4d388932.bundle.js:30
n @ vendor-4d388932.bundle.js:30
O @ vendor-4d388932.bundle.js:30
y @ vendor-4d388932.bundle.js:30
h @ vendor-4d388932.bundle.js:30
enqueueSetState @ vendor-4d388932.bundle.js:30
i.setState @ vendor-4d388932.bundle.js:23
a @ app-4d388932.bundle.js:23
a @ app-4d388932.bundle.js:19
a.emit @ app-4d388932.bundle.js:23
t.trigger @ app-4d388932.bundle.js:19
update @ app-4d388932.bundle.js:1
(anonymous) @ app-4d388932.bundle.js:1
(anonymous) @ app-4d388932.bundle.js:6
Promise.then (async)
(anonymous) @ app-4d388932.bundle.js:6
(anonymous) @ app-4d388932.bundle.js:6
Promise.then (async)
usage @ app-4d388932.bundle.js:6
(anonymous) @ app-4d388932.bundle.js:6
(anonymous) @ app-4d388932.bundle.js:6
(anonymous) @ app-4d388932.bundle.js:6
n @ app-4d388932.bundle.js:6
(anonymous) @ app-4d388932.bundle.js:6
(anonymous) @ app-4d388932.bundle.js:6
Promise.then (async)
(anonymous) @ app-4d388932.bundle.js:6
(anonymous) @ app-4d388932.bundle.js:6
B @ app-4d388932.bundle.js:6
ia @ app-4d388932.bundle.js:6
Te @ app-4d388932.bundle.js:6
getMetadata @ app-4d388932.bundle.js:6
(anonymous) @ app-4d388932.bundle.js:6
(anonymous) @ app-4d388932.bundle.js:6
(anonymous) @ app-4d388932.bundle.js:6
```
When viewing https://openneuro.dev.sqm.io/datasets/ds001095 (renders blank screen). Could be related to #670

**Comment by chrisgorgo** [comment]
After the latest update this changed to 👍 

```
app-7f585085.bundle.js:19 Uncaught (in promise) Error: GraphQL error: Cannot destructure property `issues` of 'undefined' or 'null'.
    at new t (app-7f585085.bundle.js:19)
    at app-7f585085.bundle.js:23
    at app-7f585085.bundle.js:23
    at Array.forEach (<anonymous>)
    at app-7f585085.bundle.js:23
    at Map.forEach (<anonymous>)
    at e.broadcastQueries (app-7f585085.bundle.js:23)
    at Object.next (app-7f585085.bundle.js:23)
    at p.next (app-7f585085.bundle.js:23)
    at app-7f585085.bundle.js:23
    at Array.forEach (<anonymous>)
    at Object.next (app-7f585085.bundle.js:23)
    at c (app-7f585085.bundle.js:51)
    at p (app-7f585085.bundle.js:51)
    at e.value (app-7f585085.bundle.js:51)
    at p.next (app-7f585085.bundle.js:23)
    at app-7f585085.bundle.js:51
```
With infinite loading wheel.

**Comment by olgn** [comment]
datasets that did not run validation successfully do not have any entries in the 'summary' field of the dataset, which causes gql to fail upon attempting to parse this field into the subfields. since this dataset is incomplete, this should be resolved with the implementation of #685, which does not attempt to obtain validation data for partially uploaded datasets.
